### PR TITLE
Chart Remake - Contact Tracing

### PR DESCRIPTION
**Pull request by pnavarrc**
Review after #816 - rebase once #816 is merged

### CODE REVIEW

File: src/components/Charts/ChartContactTracing.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ChartZones from './ChartZones';
+import { CONTACT_TRACING_LEVEL_INFO_MAP } from 'common/metrics/contact_tracing';
+import { formatPercent } from './utils';
+
+const CAP_Y = 1;
+
+const getPointText = (valueY: number) => formatPercent(valueY, 0);
+
+const getTooltipText = (valueY: number) =>
```
Comment (goldblatt) [inline src/components/Charts/ChartContactTracing.tsx:10]: do we need to do the capping stuff we did in icu headroom?

File: src/components/Charts/ChartContactTracing.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  `Contacts traced ${getPointText(valueY)}`;
+
+const ChartContactTracing = ({
+  columnData,
+  height = 400,
+}: {
+  columnData: any[];
```
Comment (goldblatt) [inline src/components/Charts/ChartContactTracing.tsx:17]: also a type here would be nice

File: src/components/Charts/ChartContactTracing.tsx (hunk 1 above)
Comment (pnavarrc) [reply src/components/Charts/ChartContactTracing.tsx:10]: I'm assuming that we can't go over 100% contact tracing. For ICU headroom, there are some cases that go over 100% (I checked on the `/all` page), although I don't know how that works in real life 🤷‍♂️

File: src/components/Charts/ChartContactTracing.tsx (hunk 2 above)
Comment (pnavarrc) [reply src/components/Charts/ChartContactTracing.tsx]: Fixed!

**Comment by pnavarrc** [comment]
Fixed the missing type - merging!

**Pull request merged**